patches/sitecustomize.py

```diff
@@ -1,22 +1,56 @@
 # TODO(rosbo): Remove this once we fix the issue with fastai importing older libcudnn if imported prior to tensorflow
 import tensorflow
 import os
+from google.auth import credentials
```

Comment:
nit:  Imports in alpha order.

The `tensorflow` import is an exception to this:  The point is to always import it prior to any other imports.

So:

```
# TODO(rosbo)...
import tensorflow

from google.auth import bigquery
...
import os
```